<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>F_AjouterCotiCarte</class>
 <widget class="QWidget" name="F_AjouterCotiCarte">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>359</width>
    <height>323</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Abonnements</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <layout class="QGridLayout" name="GLay">
     <item row="5" column="2" colspan="2">
      <widget class="QLineEdit" name="LE_Duree"/>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Nom :</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Duree de validité :</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Prix</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QDateEdit" name="DtE_Expiration">
       <property name="displayFormat">
        <string>dd/MM/yyyy</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="3">
      <widget class="QLabel" name="Lb_Selection">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>Sélectionner l'abonnement :</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="3">
      <widget class="QComboBox" name="CBx_Abo">
       <item>
        <property name="text">
         <string>- - - - -</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_modif1">
       <property name="text">
        <string>Date d'expiration:</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Credit disponible :</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2" colspan="2">
      <widget class="QLineEdit" name="LE_Nom">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
      </widget>
     </item>
     <item row="7" column="2" colspan="2">
      <widget class="QLineEdit" name="LE_Prix"/>
     </item>
     <item row="8" column="2" colspan="2">
      <widget class="QLineEdit" name="LE_Credit"/>
     </item>
     <item row="6" column="1">
      <widget class="QLabel" name="Lb_Date">
       <property name="text">
        <string>Date de départ :</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2" colspan="2">
      <widget class="QDateEdit" name="DE_Depart">
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLabel" name="label_modif2">
       <property name="text">
        <string>Credit Restant:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QLineEdit" name="LE_Restant"/>
     </item>
     <item row="10" column="3">
      <widget class="QPushButton" name="Bt_Valider">
       <property name="text">
        <string>Valider</string>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="10" column="4">
      <widget class="QPushButton" name="Bt_Annuler">
       <property name="text">
        <string>Annuler</string>
       </property>
      </widget>
     </item>
     <item row="7" column="4">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>€</string>
       </property>
      </widget>
     </item>
     <item row="5" column="4">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>jours</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1" colspan="4">
      <spacer name="VS">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="1" colspan="4">
      <spacer name="VS_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
